src/views/stacks/stack-details.tsx

```diff
@@ -38,7 +47,7 @@ export const StackDetails = () => {
                 </DataTableHead>
 
                 <DataTableBody>
-                    {stack.parameters?.map((parameter) => {
+                    {sortedStack.parameters?.map((parameter) => {
```

Comment:
So here you'd do sth like `stack.parameters?.sort((a, b) => (a.priority < b.priority ? -1 : 1)).map(parameter => {`